src/index.ts

```diff
@@ -120,18 +120,23 @@ export interface QueueItem {
 
     await initDatabase(database);
 
-    const browser = await puppeteer.use(StealthPlugin()).launch({
-        executablePath: CHROME_BIN || 'google-chrome-stable',
-        args: [
-            '--no-sandbox',
+    const browserArgs = [
+        '--no-sandbox',
+
+        '--use-fake-ui-for-media-stream',
+        '--use-fake-device-for-media-stream',
 
-            '--use-fake-ui-for-media-stream',
-            '--use-fake-device-for-media-stream',
+        '--disable-web-security',
+        '--disable-features=IsolateOrigins',
+        '--disable-site-isolation-trials',
+    ];
 
-            '--disable-web-security',
-            '--disable-features=IsolateOrigins',
-            '--disable-site-isolation-trials',
-        ],
+    if (process.env.DISABLE_GPU === 'true') browserArgs.push('--disable-gpu');
+    if (process.env.DISABLE_SOFTWARE_RASTERIZER === 'true') browserArgs.push('--disable-software-rasterizer');
```

Comment:
I use another way to conditionally add elements to an array in the code, let's stick to it.
```ts
// Line 39
const {
    // [...]
	DISABLE_GPU,
} = process.env;

// [...]

const browserArgs = [
    '--no-sandbox',

    '--use-fake-ui-for-media-stream',
    '--use-fake-device-for-media-stream',

    '--disable-web-security',
    '--disable-features=IsolateOrigins',
    '--disable-site-isolation-trials',

	...(DISABLE_GPU === 'true' && ['--disable-gpu', '--disable-software-rasterizer'])
];
```

Reply:
The correct way to do this is
```Typescript
...((DISABLE_GPU === 'true' && ['--disable-gpu', '--disable-software-rasterizer']) || [])
```